src/core/Table/Table.tsx

```diff
@@ -21,6 +21,7 @@ import {
   useTable,
   ActionType,
   TableInstance,
+  useExpanded,
```

Comment:
Can you uncomment expand related stuff inside `src\types\react-table-config.ts`

Reply:
Will do. Just a question, why don't we re-export types from react-table directly?

Reply:
Too hard question 😀